src/page-view.ts

```diff
@@ -17,40 +20,45 @@ interface PageViewEventProperties {
 }
 
 export class PageViewManager {
-    _currentPath?: string
-    _prevPageviewTimestamp?: Date
+    _currentPageview?: { timestamp: Date; pageViewId: string | undefined; pathname: string | undefined }
```

Comment:
brain is not fully engaged so sorry if this is a silly question

this is entirely in memory, no? so if my app does full page loads on navigation (i.e. not a SPA) then I won't have a previous pageview id?

or am i misunderstanding?

otherwise the logic for tracking it looks valid to me

Reply:
This is true - I'll add a comment about why this is OK

Reply:
nice! thank you! 💖